SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCloudLauncher.cpp

```diff
@@ -24,38 +26,50 @@ bool SpatialGDKCloudLaunch()
 
 	if (SpatialGDKSettings->IsSimulatedPlayersEnabled())
 	{
-		LauncherCmdArguments = FString::Printf(
+		CmdArguments = FString::Printf(
 			TEXT("%s %s \"%s\" %s %s"),
-			*LauncherCmdArguments,
+			*CmdArguments,
 			*SpatialGDKSettings->GetSimulatedPlayerDeploymentName(),
 			*SpatialGDKSettings->GetSimulatedPlayerLaunchConfigPath(),
 			*SpatialGDKSettings->GetSimulatedPlayerRegionCode().ToString(),
 			*FString::FromInt(SpatialGDKSettings->GetNumberOfSimulatedPlayer())
 		);
 	}
 
-	LauncherCmdArguments = FString::Printf(
-		TEXT("%s ^& pause"),
-		*LauncherCmdArguments
-	);
+	int32 OutCode = 0;
+	FString OutString;
+	FString OutErr;
 
-	FProcHandle DeploymentLauncherProcHandle = FPlatformProcess::CreateProc(
-		*CmdExecutable, *LauncherCmdArguments, true, false, false, nullptr, 0,
-		*SpatialGDKSettings->GetDeploymentLauncherPath(), nullptr, nullptr);
+	bool bSuccess = FPlatformProcess::ExecProcess(*LauncherExe, *CmdArguments, &OutCode, &OutString, &OutErr);
+	if (OutCode != 0)
+	{
+		UE_LOG(LogSpatialGDKEditorCloudLauncher, Warning, TEXT("Cloud Launch failed: %s : %s"), *OutString, *OutErr);
+		bSuccess = false;
+	}
 
-	return DeploymentLauncherProcHandle.IsValid();
+	return bSuccess;
 }
  
 bool SpatialGDKCloudStop()
 {
-	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	UE_LOG(LogSpatialGDKEditorCloudLauncher, Warning, TEXT("Function not available"));
+	return false;
+
+	// TODO: UNR-2435 - Add a Stop Deployment button and fix the code below:
+	// get and provide deployment-id to stop the deployment as one of the CmdArguments
+	const FString CmdArguments = TEXT("stop");
+	const FString LauncherExe = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(TEXT("SpatialGDK/Binaries/ThirdParty/Improbable/Programs/DeploymentLauncher/DeploymentLauncher.exe"));
 
-	const FString CmdExecutable = TEXT("cmd.exe");
-	const FString LauncherCmdArguments = TEXT("/c DeploymentLauncher.exe stop");
+	int32 OutCode = 0;
+	FString OutString;
+	FString OutErr;
 
-	FProcHandle DeploymentLauncherProcHandle = FPlatformProcess::CreateProc(
-		*CmdExecutable, *LauncherCmdArguments, true, false, false, nullptr, 0,
-		*SpatialGDKSettings->GetDeploymentLauncherPath(), nullptr, nullptr);
+	bool bSuccess = FPlatformProcess::ExecProcess(*LauncherExe, *CmdArguments, &OutCode, &OutString, &OutErr);
+	if (OutCode != 0)
+	{
+		UE_LOG(LogSpatialGDKEditorCloudLauncher, Warning, TEXT("Cloud Launch failed: %s : %s"), *OutString, *OutErr);
```

Comment:
Let's print the `OutCode` here as well as the `OutString`

Reply:
Added.